activiti-cloud-starter-configure/src/main/java/org/activiti/spring/bpmn/parser/CloudActivityBehaviorFactory.java

```diff
@@ -0,0 +1,649 @@
+package org.activiti.spring.bpmn.parser;
```

Comment:
@daisuke-yoshimoto same as before.. let's leave this extension mechanism for after beta1

Reply:
@Salaboy 
I removed this extension mechanism.